styles/_body.scss

```diff
@@ -0,0 +1,47 @@
+.Description {
+    font-family: "Akaya Telivigala";
+    font-weight: 100;
+    text-decoration-line:underline;
+    font-size: 24px;
+    text-shadow: 1px 0 0 #000, 0-1px 0 #000, -1px 0 0 #000;
+    color: $SecondaryColor;
+    grid-area: Description;
+    justify-content: center;
+    text-align: center;
+    text-justify: auto;
+    margin: 20px 20px 20px 20px;
+  }
+
+  .Columna-Img {
+    grid-area: Columna-Img;
+    max-height: 200px;
+    max-width: 200px;
+    min-height: 100px;
+    min-width: 100px;
+    width: 200px;
+    height: 200px;
+    justify-content: flex-start;
+  }
+  
+  .Columna-Img2 {
+    grid-area: Columna-Img2;
+    max-height: 200px;
+    max-width: 200px;
+    min-height: 100px;
+    min-width: 100px;
+    width: 200px;
+    height: 200px;
+    justify-content: space-evenly;
+  }
+
+  .imag {
+    width: 200px;
+    height: 200px;
+    max-width: 1fr;
```

Comment:
the unit `fr` is only used for dividing repeating cssgrids columns / rows. replace this with `px`